default.nix

```diff
@@ -16,8 +15,10 @@
 
 let
   defaultSources = import ./nix/sources.nix;
-  pkgsDefault = import (defaultSources.nixpkgs) {};
+  pkgsDefault = import defaultSources.nixpkgs (import defaultSources."haskell.nix");
```

Comment:
I don't like this it essentially means you need haskell.nix builtin the overlay for projects that don't use haskell at all.